shimmingtoolbox/utils.py

```diff
@@ -7,6 +7,10 @@
 import tqdm
 import subprocess
 import logging
+from pathlib import Path
+
+HOME_DIR = str(Path.home())
+PATH_ST_VENV = f"{HOME_DIR}/shimming-toolbox/python/envs/st_venv/bin"
```

Comment:
Consider opening an issue to address users moving ST installation folder (we've seen that many times with our SCT users).